### PR TITLE
Scott buermann rec

### PR DESCRIPTION
**Pull request by Scott-Buermann**
Recommendations
Closes #17

### CODE REVIEW

File: Scott-Buermann-rec.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# St. Paul, MN
+
+### Places to Eat
+- Dark Horse
```
Comment (ithinkisam) [inline Scott-Buermann-rec.md]: @Scott-Buermann Any good food options here?

File: Scott-Buermann-rec.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# St. Paul, MN
```
Comment (ithinkisam) [inline Scott-Buermann-rec.md:1]: @Scott-Buermann What about in your neck of the woods?

**Review by ithinkisam — CHANGES_REQUESTED** [review]
More detail please!

File: Scott-Buermann-rec.md (hunk 2 above)
Comment (Scott-Buermann) [reply Scott-Buermann-rec.md:1]: la bomba, Space Aliens

File: Scott-Buermann-rec.md (hunk 1 above)
Comment (Scott-Buermann) [reply Scott-Buermann-rec.md]: not sure - only had HH there

**Review by Scott-Buermann — COMMENTED** [review]
@ithinkisam fixed it

**Review by ithinkisam — APPROVED** [review]
Looks good!

**Pull request merged**